components/org.wso2.carbon.identity.application.authenticator.sessionauth/src/main/java/org/wso2/carbon/identity/application/authenticator/sessionauth/internal/SessionAuthenticatorServiceComponent.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.application.authenticator.sessionauth.internal;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.wso2.carbon.identity.application.authentication.framework.ApplicationAuthenticator;
+import org.wso2.carbon.identity.application.authenticator.sessionauth.SessionCountAuthenticator;
+
+/**
+ * OSGI component for user's current active session count based authenticator.
+ */
+
+@Component(
+        name = "identity.application.authenticator.sessionauth.component",
+        immediate = true
+)
+public class SessionAuthenticatorServiceComponent {
+
+    private static Log log = LogFactory.getLog(SessionAuthenticatorServiceComponent.class);
+    @Activate
+    protected void activate(ComponentContext ctxt) {
+        try {
+            SessionCountAuthenticator sessionCountAuthenticator = new SessionCountAuthenticator();
+            ctxt.getBundleContext().registerService(ApplicationAuthenticator.class.getName(),
+                    sessionCountAuthenticator, null);
+            if (log.isDebugEnabled()) {
+                log.info("SessionCountAuthenticator bundle is activated");
```

Comment:
This has to be a debug log.

Reply:
Updated in this commit[1] 

[1] - 00e051a26f6174048254eb7977f68757979c2e87